integration/out-of-bounds-read.test.ts

```diff
@@ -0,0 +1,81 @@
+import { describe, before, test } from 'mocha'
+import { expect } from 'chai'
+import express from 'express'
+import { CreateHttpServer } from '../src/server'
+import {
+  getOutOfBoundsReadCheri,
+  getOutOfBoundsReadAarch64,
+  getInvalidExecutable,
+} from '../integration/helper/routeHelper'
+
+describe('Tests aarch64 version', () => {
+  let app: express.Express
+
+  before(async function () {
+    app = await CreateHttpServer()
+  })
+
+  test('Happy Path - aarch64', async () => {
+    const response = await getOutOfBoundsReadAarch64(app, 'pass', -32, -28)
+
+    expect(response.status).to.equal(200)
+    expect(response.body.status).to.contain('success')
+  })
+
+  test('Bad Parameters - aarch64', async () => {
+    const response = await getOutOfBoundsReadAarch64(app, 'badpass', NaN, 'ttttttt')
+
+    expect(response.status).to.equal(200)
+    expect(response.body.status).to.contain('success')
```

Comment:
I see the response body for this test and the `Missing parameters...` test is ` 'Storing Secret...\nThe data between indices 0 and 0 is: \n'`. Is that the kind of output we expect from bad params and do we want to assert that output in these tests?